src/wp-includes/class-wp-block-styles-registry.php

```diff
@@ -60,6 +60,15 @@ public function register( $block_name, $style_properties ) {
 			return false;
 		}
 
+		if ( strpos( $style_properties['name'], ' ' ) !== false ) {
```

Comment:
Shouldn't this should use a Yoda condition?, e.g.,

`if ( false !== strpos( $style_properties['name'], ' ' ) ) {}`

Reply:
Yeah, either that or using `str_contains()` as suggested above would be my preference too.

Reply:
agreed...`str_contains()` is better and then no need for Yoda